designer/server/src/main/scala/pl/touk/nussknacker/ui/definition/UIProcessObjectsFactory.scala

```diff
@@ -176,7 +172,10 @@ object UIProcessObjectsFactory {
       details    <- fragmentsDetails
       definition <- definitionExtractor.extractFragmentComponentDefinition(details.canonical).toOption
     } yield {
-      details.canonical.id -> definition.toStaticDefinition(details.category)
+      details.canonical.id -> definition.toStaticDefinition(
+        details.category,
+        Try(details.canonical.metaData.typeSpecificData.asInstanceOf[FragmentSpecificData].docsUrl).getOrElse(None)
```

Comment:
Let's add this `docUrl` into `FragmentComponentDefinition` during component definition extraction.